src/app/shared/navbar/navbar.html

```diff
@@ -1,6 +1,11 @@
 <!-- TODO: figure out if the <nav> should go inside of a <header> element. -->
 <nav class="docs-navbar-header" aria-label="Top Toolbar"
      [class.is-next-version]="isNextVersion">
+  <div class="skip-link-wrapper" [class.cdk-visually-hidden]=skipLinkHidden>
+    <a [href]=skipLinkTarget id="skip-link" (focus)="setSkipLinkVisible()" (blur)="setSkipLinkInvisible()">
```

Comment:
nit: rather than having 2 method you could just have one that takes an argument, or 0 methods: `(focus)="skipLinkHidden = false"`

Reply:
added that plus `(blur)="skipLinkHidden = true"` otherwise the skip link would stay there